src/mixins/editMixin.js

```diff
@@ -13,10 +13,8 @@ export default {
   },
   watch: {
     value (current, previous) {
-      // we want to make sure it's _really_ changed or we risk undoing the users changes
-      if (current !== previous || !deepEqual(current, previous)) {
-        this.reset()
-      }
+      const changes = objectDiff(previous, current)
+      this.edit = cloneDeep({ ...this.edit, ...changes })
```

Comment:
Should be able to leave out the `cloneDeep` here? As the `{}` creates a new object, and I don't think we have any interesting nested data? (maybe it's not needed in the other places either...).

I guess we should get this under test coverage so can answer these questions better...

Change if you want, or keep it like this :)

Reply:
I'd leave it for now. By the way, I'm quite sure that the objectDiff doesn't work if a value gets added that is not on the top level ob the `edit` object, e.g. hypothetical `edit.location.latitude`